integration/testdata/helm-render/skaffold-helm/templates/NOTES.txt

```diff
@@ -0,0 +1 @@
+Hello Helm
```

Comment:
nit: we might not need to include this file

Reply:
yea I'll remove this since it's just a leftover from cloning the other example